linked_list/linked_list.py

```diff
@@ -9,67 +9,120 @@ def __init__(self, value, next_node = None):
 # Defines the singly linked list
 class LinkedList:
     def __init__(self):
-      self.head = None # keep the head private. Not accessible outside this class
+        self.head = None # keep the head private. Not accessible outside this class
 
     # returns the value in the first node
     # returns None if the list is empty
-    # Time Complexity: ?
-    # Space Complexity: ?
+    # Time Complexity: O(1)
+    # Space Complexity: O(n)
     def get_first(self):
-        pass
+        if self.head == None:
+            return None
+        else: 
+            return self.head.value
 
 
     # method to add a new node with the specific data value in the linked list
     # insert the new node at the beginning of the linked list
-    # Time Complexity: ?
-    # Space Complexity: ?
+    # Time Complexity: O(1)
+    # Space Complexity: O(n)
     def add_first(self, value):

```

Comment:
👍  Also O(1) for space complexity since the method only ever adds 1 node to the list.